soapui-maven-plugin/project.xml

```diff
@@ -262,9 +272,9 @@
 			<type>jar</type>
 		</dependency>
 		<dependency>
-			<groupId>bouncycastle</groupId>
-			<artifactId>bcprov</artifactId>
-			<version>jdk15-144</version>
+			<groupId>org.bouncycastle</groupId>
+			<artifactId>bcprov-jdk15on</artifactId>
+			<version>1.64</version>

```

Comment:
you can add bouncycastle in the root pom with the specified version, and you won't need to specify version in child projects then.